custom_components/adaptive_lighting/strings.json

```diff
@@ -44,7 +44,8 @@
           "take_over_control": "take_over_control: If anything but Adaptive Lighting calls 'light.turn_on' when a light is already on, stop adapting that light until it (or the switch) toggles off -> on.",
           "detect_non_ha_changes": "detect_non_ha_changes: detects all >10% changes made to the lights (also outside of HA), requires 'take_over_control' to be enabled (calls 'homeassistant.update_entity' every 'interval'!)",
           "transition": "Transition time when applying a change to the lights (seconds)",
-          "adapt_delay": "adapt_delay: wait time between light turn on (seconds), and Adaptive Lights applying changes to the light state. May avoid flickering."
+          "adapt_delay": "adapt_delay: wait time between light turn on (seconds), and Adaptive Lights applying changes to the light state. May avoid flickering.",
+		  "dim to warm": "dim_to_warm: Adjust color temperature even further on a parabolic curve depending on the current brightness of the light. Does not work with take_over_control"
```

Comment:
there seem to be some tabs mixed in

Reply:
The fact that it does not work with `take_over_control` seems problematic. Any fundamental reason why not?

Reply:
No, it should work with all config options.

Reply:
Fixed